mage_integrations/mage_integrations/destinations/redshift/__init__.py

```diff
@@ -63,6 +65,14 @@ def build_create_table_commands(
                 unique_constraints=unique_constraints,
                 use_lowercase=self.use_lowercase,
             ),
+            build_create_table_command(
+                column_type_mapping=self.column_type_mapping(schema),
+                columns=schema['properties'].keys(),
+                full_table_name=temp_table_name,
+                if_not_exists=True,
+                unique_constraints=unique_constraints,
+                use_lowercase=self.use_lowercase,
+            )
```

Comment:
do we have any logic to delete this temporary table?

Reply:
No actually, i guess we don't have should we add this ?

Reply:
We should if we are going to be creating it every sync

Reply:
actually currently it's tried if table not exists only

Reply:
I see. I still think we should delete it at the end of the sync to avoid leaving a temp table in the database.